wallet/src/account/output_cache/mod.rs

```diff
@@ -1401,83 +1485,108 @@ impl OutputCache {
         })
     }
 
-    /// Mark a transaction and its descendants as abandoned
-    /// Returns a Vec of the transaction Ids that have been abandoned
-    pub fn abandon_transaction(
+    // Removes a tx from unconfirmed descendant.
+    // Returns provided tx and all the descendants.
+    fn remove_from_unconfirmed_descendants(
```

Comment:
The other thing that its caller depends on is the fact that children always come after the parents in the result. It makes sense to mention this as well.